__tests__/actions/executeFetch.spec.js

```diff
@@ -193,19 +279,19 @@ describe('executeFetch', () => {
         { status: 200, headers: { 'Content-Type': 'application/json' } }
       );
       const thunk = executeFetch({
-        resource, id, opts: { ...opts, method: 'POST' }, actionType: 'LOAD',
+        resource, id, opts: { ...opts, method: 'POST' }, actionType: 'CREATE',
```

Comment:
This is already the default method for `CREATE`, can you please change the overridden method to differ from 'POST'? The assertions below are still correct.

Reply:
good catch. addressed